scripts/pr-audit.sh

```
@@ -0,0 +1,934 @@
+PR=1
+QUICK=false
+FIX_STALE=false
+AUTO_RESOLVE=false
+JSON_OUTPUT=false
+VERIFY_RESOLVED=false
+RETRIGGER=false
+
+for arg in "$@"; do
+  case "$arg" in
+    --quick) QUICK=true ;;
+    --fix-stale) FIX_STALE=true ;;
```

Shellcheck (0.11.0)

[warning] FIX_STALE appears unused. Verify use (or export if used externally).

(SC2034)